common/src/main/java/net/william278/huskhomes/command/TpCommand.java

```diff
@@ -88,9 +87,23 @@ private void execute(@NotNull CommandUser executor, @NotNull Teleportable telepo
         final TeleportBuilder builder = Teleport.builder(plugin)
                 .teleporter(teleportable)
                 .target(target);
-        if (executor instanceof OnlineUser user) {
-            builder.executor(user);
+
+        if (executor instanceof OnlineUser onlineUser) {
+            if (!onlineUser.hasPermission(getPermission("other"))) {
```

Comment:
Yeah, but what if the `executor` _is_ the person being teleported? We shouldn't require "other" in this case.

Reply:
I moved it into the teleportable if clause